src/Wallet/Cip30.purs

```diff
@@ -132,6 +134,14 @@ getBalance wallet = do
     bytes <- mbBytes
     fromBytes (unwrap bytes) >>= convertValue
 
+getNetworkId :: Cip30Connection -> Aff NetworkId
+getNetworkId wallet = do
+  networkId <- toAffE $ _getNetworkId wallet
+  case networkId of
+    0 -> pure TestnetId
+    1 -> pure MainnetId
```

Comment:
Are there IDs for the other networks e.g the ones you see in Nami's config?

<img width="374" alt="Screen Shot 2022-10-13 at 4 20 36 PM" src="https://user-images.githubusercontent.com/98818692/195703047-85658461-fc55-4c87-bcd2-3a5f50e5e27e.png">

Reply:
Network ID /= network magic

Completely different concepts. It's TestnetId for all three of preprod, preview and testnet. It affects the bech32 prefix that is used to form addresses.

See https://github.com/input-output-hk/cardano-ledger/blob/01b598bbed3626a34792d6452dcb565ddc357b33/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L384